taworks/taform/templates/taform/application.html

```diff
@@ -191,18 +192,18 @@ <h2>Course Application</h2>
 {% endfor %}
 
 <h2>Confirmation</h2>
-<div id="divWordWrap">
+<div class="divWordWrap">
     I fully understand that I can only submit one application per academic term, 
     and that once the application is submitted, no changes or modifications of 
     any sort can be done to my application.
 </div><br>
-<div id="divWordWrap">
+<div class="divWordWrap">
```

Comment:
can we make this word wrap longer? confirmation looks weird
![screen shot 2018-03-06 at 11 43 17 am](https://user-images.githubusercontent.com/22120696/37045536-24f0fdd0-2134-11e8-97b7-114fb0b52083.png)